statsmodels/survey/tests/test_summary_stats.py

```diff
@@ -0,0 +1,18 @@
+import summary_stats
+import unittest
+import pandas as pd
+import numpy as np
+
+# df = pd.read_csv("examples/survey_df.csv")
```

Comment:
Start with a test for SurveyDesign for a very small example, e.g. here:

https://gist.github.com/kshedden/bed77788173c6bde59a0db1f0b3c8d9d